### PR TITLE
Add Function (Save Data, Send & Receive Data) 

### LINKED ISSUE #4: Save data using Room
We're going to send data to the listener using euphony speaker.
in the process of transmission, We use Room as a means to save data.

### LINKED ISSUE #5: Send and Receive using euphony
We're going to send business card's data using euphony.

### PR DESCRIPTION
**Pull request by psh2849**
**1. Result**
1) Send My Card

2) Save another people's card
result

**2. How to use**
1) If you want to send a business card, fill in the contents of your business card and click the airplane button.
Then, you can speak to another!

 
2) When the listener receives your business card, it looks like this:

**Log :** 
startListen

**In the Card Box:**
listenerResult2

### CODE REVIEW

**Review by seohy02 — COMMENTED** [review]
I cloned and built this repository.
There's no error when building, but the application isn't working.....
Does it need extra settings?  Did the others build it successfully? Thanks !
image

**Comment by psh2849** [comment]
> I cloned and built this repository.
> There's no error when building, but the application isn't working.....
> Does it need extra settings? Did the others build it successfully? Thanks !
> image

Thank you for your review!
I checked this problem, I found my fault.,

You can check this problem at [res - raw - paper_plane.json]

- Before
image

- After
image

Input '{' to first Line

**Comment by Jsueeee** [comment]
> > I cloned and built this repository.
> > There's no error when building, but the application isn't working.....
> > Does it need extra settings? Did the others build it successfully? Thanks !
> > image
> 
> Thank you for your review!
> I check this problem, I found my fault.,
> 
> You can check this problem at [res - raw - paper_plane.json]
> 
> * Before
>   image
> * After
>   image
> 
> Input '{' to first Line

I also had the same error, but applying this, solved it!
Thanks!

File: app/src/main/java/com/recipe/android/hereismycard/makeCard/MakeCardFragment.kt [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+
+    private fun makeCardData() {
+        mRxManager.acousticSensor = AcousticSensor { card ->
+            Log.i("RESULT_TEXT", card)
+            val split = card.split("/")
+            val name = split[0]
+            val tel = split[1]
+            val email = split[2]
+            val address = split[3]
+            val bgPosition = split[4].toInt()
```
Comment (Jsueeee) [inline app/src/main/java/com/recipe/android/hereismycard/makeCard/MakeCardFragment.kt]: If you use this not-null string, 
the app crashes as shown in this picture.

image

We also need to consider the case when the sender does not fill in the blanks, 
so why not fix it like the one below?

```suggestion
            var name = ""
            var tel = ""
            var email = ""
            var address = ""
            var bgPosition = 0
            card.split("/").forEachIndexed { idx, split ->
                when (idx) {
                    0 -> name = split
                    1 -> tel = split
                    2 -> email = split
                    3 -> address = split
                    4 -> bgPosition = split.toInt()
                }
            }
```

**Comment by seohy02** [comment]
> > I cloned and built this repository.
> > There's no error when building, but the application isn't working.....
> > Does it need extra settings? Did the others build it successfully? Thanks !
> > image
> 
> Thank you for your review!
> I checked this problem, I found my fault.,
> 
> You can check this problem at [res - raw - paper_plane.json]
> 
> * Before
>   image
> * After
>   image
> 
> Input '{' to first Line

It solved !! Thanks!!

File: app/src/main/java/com/recipe/android/hereismycard/makeCard/MakeCardFragment.kt (hunk 1 above)
Comment (psh2849) [reply app/src/main/java/com/recipe/android/hereismycard/makeCard/MakeCardFragment.kt]: It works well! 
Thanks!

**Review by seohy02 — APPROVED** [review]
You have done a great job !!!!

**Pull request merged**